### ISSUE TITLE
Discussion Forum Shadows on Dark Mode

### ISSUE THREAD

**Issue by jputlock** [Submitty/Submitty#5874]
**What problem are you trying to solve with Submitty**
When using the normal color mode in Submitty, it uses shadows to visibly distinguish comments from each other. However, when using one of the Dark modes (normal or black), it removes these shadows (or they aren't visible) and the comments are no longer visibly separated.

**Describe the way you'd like to solve this problem**
Adding an outline to the comment boxes in dark mode (probably light grey or grey depending on which mode) to visibly separate the comment boxes.

Example:
![darkmode](https://user-images.githubusercontent.com/8323852/92293847-9eb03380-eef4-11ea-9fac-f2fe9d5bf69f.png)
![lightmode](https://user-images.githubusercontent.com/8323852/92293848-9eb03380-eef4-11ea-9271-308a1dae918d.png)
